setup.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# Copyright (c) 2016 Tigera, Inc. All rights reserved.
+#
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import setuptools
+
+version = '0.9.0'
```

Comment:
I think this should be 0.9.0.dev1 or similar (that's in keeping with our master version  convention of Felix.